test/cli/trailing_commas_test.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:dart_style/src/dart_formatter.dart';
+import 'package:test/test.dart';
+import 'package:test_descriptor/test_descriptor.dart' as d;
+
+import '../utils.dart';
+
+void main() {
+  compileFormatter();
```

Comment:
[optional] Consider using `setUpAll(compileFormatter);` to avoid extra work when the suite is loaded without running any test cases, such as test sharding.

Reply:
The body of `compileFormatter()` itself calls `setUpAll()`. :)